net/shorewall-lite/Makefile

```diff
@@ -58,19 +57,19 @@ define Build/Compile
 endef
 
 define Package/shorewall-lite/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(INSTALL_DIR) $(1)/etc/shorewall-lite/state
-	$(INSTALL_DIR) $(1)/usr/share
+	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface/
```

Comment:
Not sure I see the point of changing the order unnecessarily.

Reply:
One of the reviewers suggested an alphabetic order. Not necessary but would be nice.

Reply:
I prefer smaller change sets/low touch.

Reply:
I understand. Is this a showstopper and have I to revert it?
Personally I find an ordered list easier readable.

No problem reverting it if it is a no-go.

Reply:
Please restore the original ordering (and hence the smallest possible patchest).

Patches with superfluous changes are a lot more work to back out if they ever need to be reverted.

Reply:
Done.

Will make a PR after this one.